website/docs/patterns/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Patterns
+status: comingSoon
+---
+
+## What is a pattern?
+
+Pattern generally refers to a combination of components which work together to accomplish a function or render a layout and is often used to solve the same problem in a consistent manner. Common examples of patterns are forms, navigation, combinations of action buttons, etc.
+
+> "Components are an interface chunk to be added to an overall layout. whereas patterns may be UI or a variety of other things, like a behavior, flow, application motif or something else." ([Nathan Curtis](https://medium.com/eightshapes-llc/patterns-components-2ce778cbe4e8))
+
+## HDS pattern support
+
+Patterns are not currently supported by the HashiCorp Design System, but there are future plans to introduce guidelines, documentation, and best practices around assembling HDS components in a consistent and scalable manner.
+
+!!! Info
+
+**Note:** 
+
+There are a small number of HDS components that showcase a pattern-like level of complexity ([Modal](/components/modal), [Table](/components/table), [Pagination](/components/pagination)) but are considered components and published as such.
+!!!
+
+If you find yourself building the same type of pattern frequently using HDS components, consider [contributing](/getting-started-contribution) back to the HashiCorp Design System and making your work available to other designers and developers!
```

Comment:
We don't accept pattern contributions at this time. This is great for the future, but we should keep a consistent message about what contributions are accepted. Maybe instead, the call to action is to submit a request.

Reply:
Ok, I'll pivot the language to point towards submitting a request

Also thinking on it, does it make sense to include something around reaching out to us for suggestions and best-practices when building local components (referencing our earlier thread about local components within the flyout)?

Reply:
Ohh, good question. I know DesignOps is working on securing some educational resources that may help, but it couldn't hurt to let folks know we're happy to help. 🙂